packages/cli/configs/ttk/basic/csharp/TeamsApp/.gitignore

```diff
@@ -0,0 +1,10 @@
+# TeamsFx files
+build
+appPackage/build
+env/.env.*.user
+env/.env.local
+appsettings.Development.json
+.deployment
```

Comment:
Should we adjust the directory name / feature name from ttk since these files are being touched? Or make the breaking change later?

Reply:
will make these fixes as a fast follow